kong/db/schema/entities/cas.lua

```diff
@@ -0,0 +1,41 @@
+local typedefs = require "kong.db.schema.typedefs"
+local openssl_x509 = require "openssl.x509"
+local ngx_time = ngx.time
+
+return {
+  name        = "cas",
+  primary_key = { "id" },
+
+  fields = {
+    { id             = typedefs.uuid, },
+    { created_at     = typedefs.auto_timestamp_s },
+    { cert           = typedefs.certificate { required = true }, },
+    { tags           = typedefs.tags },
```

Comment:
style: we find it to be easier not to align `=` signs anymore, if you care to revert the change... Not a blocker for me.

Reply:
This has been reverted.